extensions/resteasy-reactive/quarkus-resteasy-reactive/deployment/src/main/resources/dev-templates/view.html

```diff
@@ -1,4 +1,22 @@
 {#include main}
 {#title}Quarkus Static Resources{/title}
-
+{#style}
+{/style}
+{#body}
+{#let resource=currentRequest.params.get("path")}
+{#for childOfPath in info:staticResourceInfo.resourceMap.get(resource).children}
+<div class="row">
+    <div class="col">
+        {#if !childOfPath.isFolder}
+            <i class="fas fa-file-alt" style="margin-right:5px; margin-top:5px"></i>
+            {childOfPath.lastName}
+        {#else}
+            <i class="far fa-folder-open" style="margin-right:5px; margin-top:5px"></i>
+            {childOfPath.lastName}
+        {/if}
+    </div>
+</div>
+{/for}
+{/let}
+{/body}
```

Comment:
So with this commit the page now looks like this :
![image](https://user-images.githubusercontent.com/46818757/107523879-34a26580-6bdb-11eb-8b0a-1ede3466dd74.png)

**On clicking `index.html` :**
![image](https://user-images.githubusercontent.com/46818757/107523969-4ab02600-6bdb-11eb-9328-b7007e61749a.png)

**On clicking `/Images` :**
![image](https://user-images.githubusercontent.com/46818757/107524070-64ea0400-6bdb-11eb-937c-1993eab5c3ba.png)

Inside `view.html` , will separate `Images/icon.png` to list file names without their path i.e only `icon.png` as you suggested :)
maybe in a sep PR or in this one only. 

Thanks